Merge Skinned Mesh Enablement Animation (#1173)
anatawa12/AvatarOptimizer: .docs/content/docs/reference/merge-skinned-mesh/index.md

If this checkbox is checked, renderers whose enablement is different than target renderer on the build time will not be merged.

### Copy Enablement Animation

> If this checkbox is checked, the activeness / enablement animation of the target renderer will be copied to the merged renderer.

Author: source => destinationで書いてるからsourceでも間違いではないというかmerge sourceって最初の方は書いてた気がする

> If this checkbox is checked, the activeness / enablement animation of the target renderer will be copied to the merged renderer.

Reviewer: docsはほぼtargetで書いてるかも(どこかのタイミングで私が書き換えてしまったかも?)

This feature may copy animation animating `enabled` of the target renderer or `activeSelf` of the GameObject or ancestor GameObjects.

> This feature supports copying only one animated property so if there are multiple animated properties (in other words, both `enabled` and `activeSelf`, or multiple `activeSelf` are animated), it will be error.

Author: ```
common Root
 `- Parent 1
     `-Parent 2
         `- Renderer 
```

とあったときにParent 1とParent 2の療法がアニメーションされてるとだめです

> This feature supports copying only one animated property so if there are multiple animated properties (in other words, both `enabled` and `activeSelf`, or multiple `activeSelf` are animated), it will be error.

Reviewer: ```suggestion
This feature supports copying only one animated property so if there are multiple animated properties (e.g., both `enabled` and `activeSelf` are animated, or `activeSelf` are animated on multiple layers), it will be error.
```
とかどうでしょう

> This feature supports copying only one animated property so if there are multiple animated properties (in other words, both `enabled` and `activeSelf`, or multiple `activeSelf` are animated), it will be error.

Reviewer: 両方がアニメーションされてる場合などが駄目なのは、それが全体で共通かどうかを判定するのが大変だからですかね?

> This feature supports copying only one animated property so if there are multiple animated properties (in other words, both `enabled` and `activeSelf`, or multiple `activeSelf` are animated), it will be error.

Author: 両方がアニメーションされている場合には、どちらもtrueなときにMerged SMRのenabledをtrueにする(どちらかがfalseならfalseにする)という面倒なアニメーション(or blendtree)を組まないといけないですが、それの生成が難しい(面倒)なために制限を設けてます。

一つだけならアニメーションをコピペするだけで済む(そしてそのためのインフラがAAOにはある)ので

> This feature supports copying only one animated property so if there are multiple animated properties (in other words, both `enabled` and `activeSelf`, or multiple `activeSelf` are animated), it will be error.

Author: > `activeSelf` are animated on multiple layers

animator layerと勘違いされるとおもう。

> This feature supports copying only one animated property so if there are multiple animated properties (in other words, both `enabled` and `activeSelf`, or multiple `activeSelf` are animated), it will be error.

Reviewer: それはちょっと思った
階層を表現する方法が分からなかった(複数というだけだと階層の意味は認識しづらいと思っています)

> This feature supports copying only one animated property so if there are multiple animated properties (in other words, both `enabled` and `activeSelf`, or multiple `activeSelf` are animated), it will be error.

Author: both one `activeSelf` and parents' `activeSelf` とか？

> This feature supports copying only one animated property so if there are multiple animated properties (in other words, both `enabled` and `activeSelf`, or multiple `activeSelf` are animated), it will be error.

Reviewer: いいかも

> This feature supports copying only one animated property so if there are multiple animated properties (in other words, both `enabled` and `activeSelf`, or multiple `activeSelf` are animated), it will be error.

Author: じゃあ日本語も同じ方針でお願いします

> This feature supports copying only one animated property so if there are multiple animated properties (in other words, both `enabled` and `activeSelf`, or multiple `activeSelf` are animated), it will be error.

Author: ```suggestion
This feature supports copying only one animated property so if there are multiple animated properties (e.g., both `enabled` and `activeSelf` are animated, or both one `activeSelf` and parents' `activeSelf`), it will be error.
```

> This feature supports copying only one animated property so if there are multiple animated properties (e.g., both `enabled` and `activeSelf` are animated, or both one `activeSelf` and parents' `activeSelf`), it will be error.

Reviewer: ```suggestion
This feature supports copying only one animated property so if there are multiple animated properties (e.g., both `enabled` and `activeSelf` are animated, or both one `activeSelf` and parents' `activeSelf` are), it will be error.
```
